lib/scala/edition-updater/src/main/scala/org/enso/editions/updater/EditionManager.scala

```diff
@@ -79,6 +87,6 @@ object EditionManager {
     distributionManager: DistributionManager,
     languageHome: Option[LanguageHome] = None
   ): EditionManager = new EditionManager(
-    makeEditionProvider(distributionManager, languageHome)
+    makeEditionProvider(distributionManager, languageHome, false)
```

Comment:
This [little change](https://github.com/enso-org/enso/pull/10374/commits/f1ad21b09c481bdb5f6f72d62cf48e4f740ccd45) is contribution to
- #9231

e.g. don't download anything.

Reply:
- reported as separate #10408